### PR TITLE
Remove resources post test

### PR DESCRIPTION
**Pull request by seankane-msft**

### CODE REVIEW

**Comment by seankane-msft** [comment]
Addresses #12764

**Comment by seankane-msft** [comment]
And #13406

**Comment by kristapratico** [comment]
The "packing update" commit seems to have done something strange to this PR...

File: sdk/tables/azure-data-tables/README.md [hunk 1]
```diff
@@ -1,299 +1,21 @@
-# Azure Data Tables client library for Python
+# Microsoft Azure SDK for Python
```
Comment (iscai-msft) [inline sdk/tables/azure-data-tables/README.md:1]: readme shouldn't be changed

File: sdk/tables/azure-data-tables/setup.py [hunk 2]
```diff
@@ -67,18 +74,17 @@
...
         # Exclude packages that will be covered by PEP420 or nspkg
         'azure',
-        'tests',
+        'azure.data',
     ]),
     install_requires=[
-        "azure-core<2.0.0,>=1.2.2",
-        "msrest>=0.6.10"
-        # azure-data-tables
+        'msrest>=0.5.0',
```
Comment (iscai-msft) [inline sdk/tables/azure-data-tables/setup.py:83]: assuming lower dependency should stay the same. You should also still need azure-core dependency

File: sdk/tables/azure-data-tables/setup.py [hunk 3]
```diff
@@ -67,18 +74,17 @@
...
+        'msrestazure>=0.4.32,<2.0.0',
+        'azure-common~=1.1',
     ],
     extras_require={
-        ":python_version<'3.0'": ['futures'],
-        ":python_version<'3.4'": ['enum34>=1.0.4'],
-        ":python_version<'3.5'": ["typing"]
-    },
+        ":python_version<'3.0'": ['azure-data-nspkg'],
```
Comment (iscai-msft) [inline sdk/tables/azure-data-tables/setup.py:88]: I feel like these extra requires should stay

File: sdk/tables/azure-data-tables/setup.py [hunk 4]
```diff
@@ -67,18 +74,17 @@
...
         'azure',
-        'tests',
+        'azure.data',
     ]),
     install_requires=[
-        "azure-core<2.0.0,>=1.2.2",
-        "msrest>=0.6.10"
-        # azure-data-tables
+        'msrest>=0.5.0',
+        'msrestazure>=0.4.32,<2.0.0',
```
Comment (iscai-msft) [inline sdk/tables/azure-data-tables/setup.py:84]: what is the dependency on msrest azure for>

File: sdk/tables/azure-data-tables/setup.py [hunk 5]
```diff
@@ -9,11 +9,11 @@
 import re
 import os.path
 from io import open
-from setuptools import find_packages, setup  # type: ignore
+from setuptools import find_packages, setup
 
 # Change the PACKAGE_NAME only to change folder and different name
 PACKAGE_NAME = "azure-data-tables"
-PACKAGE_PPRINT_NAME = "Azure Data Tables"
+PACKAGE_PPRINT_NAME = "MyService Management"
```
Comment (iscai-msft) [inline sdk/tables/azure-data-tables/setup.py:16]: Should still be Azure Data Tables

**Pull request merged**